volatility3/framework/plugins/mac/pslist.py

```diff
@@ -2,13 +2,15 @@
 # which is available at https://www.volatilityfoundation.org/license/vsl-v1.0
 #
 
+import datetime
 import logging
 from typing import Callable, Iterable, List, Dict
 
 from volatility3.framework import renderers, interfaces, exceptions
 from volatility3.framework.configuration import requirements
 from volatility3.framework.objects import utility
 from volatility3.framework.symbols import mac
+from volatility3.framework.renderers import format_hints
```

Comment:
It's a NIT comment. It is better to list modules in code in [alphabetical order](https://stackoverflow.com/questions/20762662/whats-the-correct-way-to-sort-python-import-x-and-from-x-import-y-statement).

Reply:
"Resolved: Alphabetized the imports."